test/components/Timeline/EventTest.js

```diff
@@ -55,6 +55,18 @@ describe('Event', function test () {
     })
   })
 
+  describe('#animationDelayAsSecondsString', () => {
+    it('returns the appropriate number of seconds', () => {
+      const mockEvent = {
+        timeReceived: DateTime.utc().minus(Duration.fromObject({ seconds: 9 }))
+      }
+
+      const actualString = animationDelayAsSecondsString(mockEvent)
+      expect(parseInt(actualString.slice(0, -1))).to.be.within(-12, -9)
```

Comment:
😕 Why was this change necessary? This function shouldn't take three seconds to run, can we restrict the range of acceptable calculated times to be between -9.01s and -9s ago?

`parseInt` will coerce a string beginning with a number to that number without regard to what it ends with, so `slice` is not needed and decreases readability.

Reply:
Regarding

```
expect(parseInt(actualString)).to.be.closeTo(-9, 0.001)
```

`parseInt` will truncate the decimal values, so 9.001 and 9.999 will always be 9. So the `closeTo 0.001` isn't doing much.

Regarding the time, yes, I can constraint it more, will decrease to 2 seconds. But keep in mind this is not a performance test, so it makes sense to allow some room for latency.

Regarding the `slice`, I prefer to not send the 's' character to the parseInt, so I'll keep that there.

Regarding

```
expect(actualString.endsWith('s')).to.be
```

`endsWith` does look more readable, but seems to be missing the `true` at the end. Will make that change and push.

Reply:
> `parseInt` will truncate the decimal values, so 9.001 and 9.999 will always be 9. So the closeTo 0.001 isn't doing much.

>_<